File: screen_brightness_windows/windows/src/application_screen_brightness_change_stream_handler.cpp
#include "../include/screen_brightness_windows/application_screen_brightness_change_stream_handler.h"

namespace screen_brightness
{
	void ApplicationScreenBrightnessChangeStreamHandler::AddApplicationScreenBrightnessToEventSink(double applicationScreenBrightness) const
	{
		if (sink_ == nullptr) {
			return;
		}

		sink_->Success(applicationScreenBrightness);
	}
}

File: screen_brightness_ios/ios/Classes/StreamHandler/ApplicationScreenBrightnessChangeStreamHandler.swift
//
//  ScreenBrightnessChangeEventHandler.swift
//  screen_brightness
//
//  Created by Jack on 25/10/2021.
//

import Foundation
import Flutter
import UIKit

public class ApplicationScreenBrightnessChangeStreamHandler: BaseStreamHandler {
    public func addApplicationScreenBrightnessToEventSink(_ applicationScreenBrightness: CGFloat) {
        guard let eventSink = eventSink else {
            return
        }
        
        eventSink(Double(applicationScreenBrightness))
    }
}
